js/app.js

```diff
@@ -3,15 +3,23 @@ $(function () {
   const model = new DinnerModel();
 
   // We instantiate our general controller
+  // @ts-ignore
```

Comment:
@Envl, it would indeed be nice if you could make sure that these comments do not spread over all of our JS files.